moto/settings.py

```diff
@@ -55,6 +58,7 @@ def moto_server_port():
     return os.environ.get("MOTO_PORT") or "5000"
 
 
+@lru_cache()
```

Comment:
GithubActions would sometimes take 15 seconds to make the CURL call to find the docker host.
Caching this at least ensures it only has to happen once per system